### DISCUSSION TITLE
Internal Chat

### DISCUSSION THREAD

**Discussion by chandlercoven** [chatwoot/chatwoot#5121] [Feature Requests]
Does anyone know if Chatwoot has the functionality of an internal chat between agents? So that agents can message each other internally for collaboration? The only function that I am seeing is the private note with the @ to another agent for collaborating on a specific customer. It would seem that this would also serve as a great internal chat tool but I don't see any ability to do that.

**Comment by sojan-official** [comment]
@chandlercoven Currently, Chatwoot doesn't support internal chat apart from private notes with `@`

**Reply by iamdagy** [reply]
I agree, this would be a way to improve collaboration for the support team, or the internal team

**Comment by iamdagy** [comment]
@chandlercoven did you find a workaround to mange internal chat ?

**Reply by bishboi** [reply]
just create a API channel with any webhook url and create a contact named internal communication, now send messages to this contact through the API channel

**Reply by mbdali** [reply]
is there any other way then this.. like choosing agent as contact and sending chat message or document forwarding from whatsapp to internal
